Description: Enable systemd-logind.service by default
 Start systemd-logind before starting lightdm,
 in this way, session errors will not occur,
 thus solving the problem, and also speeding
 up the startup time of lightdm.
Author: Chengyi Zhao <zhaochengyi@uniontech.com>

Origin: https://gerrit.uniontech.com/plugins/gitiles/base/systemd/+/5ca7682f777d165856153bd9a88d3adf1c77e939
Bug: https://pms.uniontech.com/zentao/bug-view-66176.html
Last-Update: 2022-05-27

--- systemd-250.4.orig/units/systemd-logind.service.in
+++ systemd-250.4/units/systemd-logind.service.in
@@ -23,6 +23,8 @@ ConditionPathExists=|/lib/systemd/system
 Wants=dbus.socket
 After=dbus.socket
 
+Before=network.target
+
 [Service]
 BusName=org.freedesktop.login1
 CapabilityBoundingSet=CAP_SYS_ADMIN CAP_MAC_ADMIN CAP_AUDIT_CONTROL CAP_CHOWN CAP_DAC_READ_SEARCH CAP_DAC_OVERRIDE CAP_FOWNER CAP_SYS_TTY_CONFIG CAP_LINUX_IMMUTABLE
@@ -65,3 +67,6 @@ SystemCallFilter=@system-service
 # Increase the default a bit in order to allow many simultaneous logins since
 # we keep one fd open per session.
 LimitNOFILE={{HIGH_RLIMIT_NOFILE}}
+
+[Install]
+WantedBy=multi-user.target
